src/exllama_predictor.py

```diff
@@ -74,10 +74,10 @@ def __init__(self, model_directory, fused_attn = True):
         self.generator = begin(generator)
     
 
-    def load_lora(self, lora_path: str) -> None:
-        # add a cache here 
+    def load_lora(self, lora_path: str) -> ExLlamaLora:
         lora = ExLlamaLora(self.model, f"{lora_path}/adapter_config.json", f"{lora_path}/adapter_model.bin")
-        self.generator.lora = lora
+        #self.generator.lora = lora
```

Comment:
is this not needed?

Reply:
it happens on L106 of predict.py

```
    def initialize_peft(self, replicate_weights: str) -> None:
        if self.current_path != replicate_weights:
            self.generator.lora = self.get_lora(replicate_weights)
```

this configuration is pretty silly but I'm not sure if I have better ideas. predict.Predictor caches loras, but creating a lora requires exllama_predictor.ExllamaGenerator.model